README.md

```diff
@@ -44,3 +44,23 @@ It is possible to use a `substra-backend` skaffold file as the source for the ne
 ```
 SUBSTRA_TESTS_SKAFFOLD_FILEPATH=$SUBSTRA_SOURCE/substra-backend/skaffold.yaml make test
 ```
+
+# Test design guidelines
+
+When adding or modifying tests, please follow these guidelines.
+
+The tests suite:
+- Must be easy to configure through settings files to define the substra network to target
+- Must be independent from the substra network
+  - Substra network must be started separately
+  - Substra network can be located locally / on the cloud
+  - Substra network can be started through docker-compose or through kubernetes
+- Requires a setup with at least 2 organisations
```

Comment:
Requires => Should target ?

Reply:
Yes.
Changed to `Should target a substra network` to be consistant with other guidelines.